frontend/src/components/QuestionInput/QuestionInput.test.tsx

```diff
@@ -276,6 +284,36 @@ describe("Test error alerts", () => {
     expect(await axe(container)).toHaveNoViolations();
   });
 
+  it("removes the error alert when the alert times out", async () => {
+    const uploadedFile = createMockFile(
+      "png",
+      ACCEPTED_FILE_TYPES.PNG,
+      MAX_UPLOADED_IMAGE_SIZE_IN_MB + 5
+    );
+
+    const { container } = render(
+      <QuestionInput
+        onSend={() => {}}
+        disabled={false}
+        placeholder={"placeholder"}
+        conversationId={undefined}
+        clearOnSend={false}
+      />
+    );
+
+    await uploadFiles([uploadedFile]);
+
+    const inputError = screen.getByTestId(`errorAlert-exceedsMaxSize-${defaultMockUuid}`);
+    expect(inputError).toBeInTheDocument();
+
+    const inputErrorAfter = screen.queryByTestId(`errorAlert-invalidFiletype-${defaultMockUuid}`);
+
+    await new Promise((_) => setTimeout(_, ERROR_ALERT_TIMEOUT_PERIOD_IN_MS));
```

Comment:
Dust: might worth being explicit about how this neat trick works
```suggestion
    await new Promise((resolve) => setTimeout(resolve, ERROR_ALERT_TIMEOUT_PERIOD_IN_MS));
```